packages/cli/src/lib/commands/generate-command.ts

```diff
@@ -150,7 +160,7 @@ export class GenerateCommand extends BaseCommand {
 
     const supportedSections = await this.generateDocumentationUseCase.getSupportedSections({
       cicdPlatform: thisCommand.getOptionValue('cicd'),
-      source: thisCommand.getOptionValue('source'),
+      source: source,
     });
```

Comment:
Keep the original behavior, pass the source option (string|string[]), the use case should manage that